### ISSUE TITLE
[vite] Internal server error: EPERM: operation not permitted, scandir 'E:\System Volume Information'

### ISSUE THREAD

**Issue by namelessinper** [vuejs/devtools-next#551]
### **Describe the bug**
hello !
In my  vite project with Vue and Typescript. I follow the document guide   When I add the vue-devtools-next and run the command pnpm  dev I get the error
[vite] Internal server error: EPERM: operation not permitted, scandir 'E:\System Volume Information'
  Plugin: vite:import-glob
  File: E:/code/lezhi/cdyf-dept-arco-vue/node_modules/.pnpm/vite-plugin-vue-devtools@7.3.7_rollup@2.79.1_vite@3.2.10_vue@3.4.34/node_modules/vite-plugin-vue-devtools/src/overlay.js?__vue-devtools-next-resource

> Here is my packge.json

```json
{
  "name": "arco-design-pro-vue",
  "description": "Arco Design Pro for Vue",
  "version": "1.0.0",
  "private": true,
  "author": "ArcoDesign Team",
  "license": "MIT",
  "scripts": {
    "dev": "vite --config ./config/vite.config.dev.ts",
    "build": "vite build --config ./config/vite.config.prod.ts",
    "report": "cross-env REPORT=true npm run build",
    "preview": "npm run build && vite preview --host",
    "type:check": "vue-tsc --noEmit --skipLibCheck"
  },
  "lint-staged": {
    "*.{js,ts,jsx,tsx}": [
      "prettier --write",
      "eslint --fix"
    ],
    "*.vue": [
      "stylelint --fix",
      "prettier --write",
      "eslint --fix"
    ],
    "*.{less,css}": [
      "stylelint --fix",
      "prettier --write"
    ]
  },
  "dependencies": {
    "@arco-design/web-vue": "^2.44.7",
    "@vueuse/core": "^9.3.0",
    "animate.css": "^4.1.1",
    "axios": "^0.24.0",
    "dayjs": "^1.11.5",
    "echarts": "^5.4.0",
    "echarts-gl": "^2.0.9",
    "gsap": "^3.11.5",
    "lodash": "^4.17.21",
    "lottie-web": "^5.12.0",
    "mitt": "^3.0.0",
    "nprogress": "^0.2.0",
    "pinia": "^2.0.23",
    "pinia-plugin-persist": "^1.0.0",
    "query-string": "^8.0.3",
    "sortablejs": "^1.15.0",
    "vue": "^3.2.40",
    "vue-draggable-next": "^2.2.1",
    "vue-echarts": "^6.2.3",
    "vue-i18n": "^9.2.2",
    "vue-router": "^4.0.14"
  },
  "devDependencies": {
    "@arco-plugins/vite-vue": "^1.4.5",
    "@commitlint/cli": "^17.1.2",
    "@commitlint/config-conventional": "^17.1.0",
    "@types/lodash": "^4.14.186",
    "@types/mockjs": "^1.0.7",
    "@types/nprogress": "^0.2.0",
    "@types/sortablejs": "^1.15.0",
    "@typescript-eslint/eslint-plugin": "^5.40.0",
    "@typescript-eslint/parser": "^5.40.0",
    "@vitejs/plugin-vue": "^3.1.2",
    "@vitejs/plugin-vue-jsx": "^2.0.1",
    "@vue/babel-plugin-jsx": "^1.1.1",
    "consola": "^2.15.3",
    "cross-env": "^7.0.3",
    "husky": "^8.0.1",
    "less": "^4.1.3",
    "lint-staged": "^13.0.3",
    "mockjs": "^1.1.0",
    "postcss-html": "^1.5.0",
    "prettier": "^2.7.1",
    "rollup": "^3.9.1",
    "rollup-plugin-visualizer": "^5.8.2",
    "typescript": "^4.8.4",
    "unplugin-vue-components": "^0.24.1",
    "vite": "^3.2.5",
    "vite-plugin-compression": "^0.5.1",
    "vite-plugin-vue-devtools": "^7.3.7",
    "vite-svg-loader": "^3.6.0",
    "vue-tsc": "^1.0.14"
  },
  "engines": {
    "node": ">=14.0.0"
  },
  "resolutions": {
    "bin-wrapper": "npm:bin-wrapper-china",
    "rollup": "^2.56.3",
    "gifsicle": "5.2.0"
  }
}

**Comment by alexzhang1030** [comment]
Looks like it's a permission-related issue, can you use admin auth to rerun `pnpm dev`?

**Comment by namelessinper** [comment]
Its solved when I updated vite to 4.5.3
It seems to be a vite3 problem
